docs/en/02_Developer_Guides/02_Controllers/02_Routing.md

```diff
@@ -154,12 +162,35 @@ This is useful when you want to provide custom actions for the mapping of `teams
 		);
 
 	    private static $url_handlers = array(
-			'staff/$ID/$Name' => 'payroll'
+			'staff/$ID/$Name' => 'payroll',
 			'coach/$ID/$Name' => 'payroll'
 	    );
 
 The syntax for the `$url_handlers` array users the same pattern matches as the `YAML` configuration rules.
 
+Now let’s consider a more complex example from a real project, where using
+**$url_handlers** is mandatory. In this example, the URLs are of the form
+`http://example.org/Feed/Go/`, followed by 5 parameters. The PHP controller
+class specifies the URL pattern in `$url_handlers`. Notice that it defines 5
+parameters.
+
+
+	:::php
+	class FeedController extends ContentController {
+
+		private static $allowed_actions = array('Go');
+		private static $url_handlers = array(
+			'Go/$UserName/$AuthToken/$Timestamp/$OutputType/$DeleteMode' => 'Go'
```

Comment:
I like the idea of a "real world" example, but this seems quite obscure and lacking context.

Could you either provide some context (perhaps as simple as adding an action) or use a concocted example which is a bit more obvious?